plugins/org.yakindu.base.xtext.utils.jface/src/org/yakindu/base/utils/jface/viewers/StyledTextCellEditor.java

```diff
@@ -115,16 +114,19 @@ private void checkSelection() {
 	/*
 	 * (non-Javadoc) Method declared on CellEditor.
 	 */
+	@Override
```

Comment:
Is this PR not only for copyright headers? 
There is a lot of other stuff in it.

Reply:
Yeah I should maybe redo this without my safe action auto formatter on.